tests/src/baseservices/typeequivalence/contracts/Types.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+[assembly:ImportedFromTypeLib("TypeEquivalenceTest")] // Required to support embeddable types
+[assembly:Guid("3B491C47-B176-4CF3-8748-F19E303F1714")]
+
+// 4D441C6F-90BB-4003-802E-2D2126E37380
```

Comment:
What are these guids?

Reply:
Those are their for future test types. I will be adding more and usually i generate a bunch in a single shot.

Reply:
It is 4 mouse clicks to generate a new Guid in VS. I do not think it makes sense to have spare guids in sources. Nobody else except will know what they are for.